src/Superdesk/ContentApiSdk/ContentApiSdk.php

```diff
@@ -174,16 +174,26 @@ private function getAssociationsFromPackage(Package $package)
         $associations = new stdClass();
 
         if (isset($package->associations)) {
-            foreach ($package->associations as $associatedName => $associatedItem) {
-                $associatedId = $this->getIdFromUri($associatedItem->uri);
+            foreach ($package->associations as $associationGroupName => $associationGroupItems) {
 
-                if ($associatedItem->type == 'composite') {
-                    $associatedObj = $this->getPackage($associatedId, true);
-                } else {
-                    $associatedObj = $this->getItem($associatedId);
+                $groupAssociations = new stdClass();
+
+                foreach ($associationGroupItems AS $associatedName => $associatedItem) {
+                    $associatedId = $this->getIdFromUri($associatedItem->uri);
+
+                    if ($associatedItem->type == 'picture') continue;
```

Comment:
I'll leave this for now, since it's only there since the picture output is still broken. Probably will remove it once it's fixed.